x/cash/codec.proto

```diff
@@ -35,6 +35,7 @@ message FeeInfo {
 
 
 message Configuration {
+  // TODO: add schema uint32 here
```

Comment:
This should land in https://github.com/iov-one/weave/issues/532 after merging. I think it's better to implement it before merging.